third_party/txt/src/txt/font_collection.cc

```diff
@@ -163,11 +163,16 @@ FontCollection::GetMinikinFontCollectionForFamilies(
     return nullptr;
   }
   if (enable_font_fallback_) {
-    for (std::string fallback_family : fallback_fonts_for_locale_[locale]) {
+    // Reverse iterate to prevent new fallback fonts changing how previously
+    // resolved glyphs are rendered.
+    auto fallback_riter = fallback_fonts_for_locale_[locale].rbegin();
+    while (fallback_riter != fallback_fonts_for_locale_[locale].rend()) {
```

Comment:
Consider the following instead. Bumping the iterator separately is sometimes error prone if this block becomes larger with conditionals etc..
```
const auto & fonts = auto fallback_fonts_for_locale_[locale];
for (auto i = fonts.rbegin(); i != rend(); i++) {
   ...
}
```